draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -0,0 +1,1568 @@
+       +--rw network-slice!
+     augment /nw:networks/nw:network:
+       +--rw (slo-sle-policy)?
+          +--:(standard)
+          |  +--rw slo-sle-template?         leafref
+          +--:(custom)
+             +--rw service-slo-sle-policy
+                +--rw policy-description?       string
+                +--rw metric-bounds
+                |  +--rw metric-bound* [metric-type]
+                |     +--rw metric-type          identityref
+                |     +--rw metric-unit          string
+
+
+
+Liu, et al.              Expires 13 August 2023                 [Page 8]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+                |     +--rw value-description?   string
+                |     +--rw bound?               uint64
+                +--rw security*                 identityref
+                +--rw isolation?                identityref
+                +--rw max-occupancy-level?      uint8
+                +--rw mtu?                      uint16
+                +--rw steering-constraints
+                |  +--rw path-constraints
+                |  +--rw service-function
+                |  +--rw disjointness?
+                |  |       te-types:te-path-disjointness
+                |  +--rw service-info?       string
+                +--rw optimization-criterion?   identityref
+                +--rw delay-tolerance?          boolean
+                +--rw periodicity*              uint64
+                +--rw resize-requirement?       identityref
+                +--rw service-info?             string
+     augment /nw:networks/nw:network/nw:node:
+       +--rw (slo-sle-policy)?
+          +--:(standard)
+          |  +--rw slo-sle-template?         leafref
+          +--:(custom)
+             +--rw service-slo-sle-policy
+                +--rw policy-description?       string
+                +--rw metric-bounds
+                |  +--rw metric-bound* [metric-type]
+                |     +--rw metric-type          identityref
+                |     +--rw metric-unit          string
+                |     +--rw value-description?   string
+                |     +--rw bound?               uint64
+                +--rw security*                 identityref
+                +--rw isolation?                identityref
+                +--rw max-occupancy-level?      uint8
+                +--rw mtu?                      uint16
+                +--rw steering-constraints
+                |  +--rw path-constraints
+                |  +--rw service-function
+                +--rw optimization-criterion?   identityref
+                +--rw delay-tolerance?          boolean
+                +--rw periodicity*              uint64
+                +--rw resize-requirement?       identityref
+                +--rw service-info?             string
+     augment /nw:networks/nw:network/nw:node/nt:termination-point:
+       +--rw sdp-id?                         leafref
```

Comment:
in my understanding, the concept of SDP is associated to the customer view. That is, a customer requests a slice asking for connectivity among SDPs, which are known by the customer. For the provider side, the NBI model is proposing peer-SAP as the counterpart of SDP.
Again in my undersatanding, what we can represent in the topology from the network (provider) to the customer is a network topology including also the peer-SAPs. But is weird to refer to peer-SAP when looking at the topology from the provider perspective.
I mean, callign something "peer-whatever" is subjective, and relative to who is the one talking (either customer or provider). So, maybe we need to define a new term for "sdp" in this context. Maybe solutions like "customer-facing-sdp", "slice-provider-sdp", etc, etc.
Keeping it as simply "sdp" I think it is confussing.

Reply:
It's better to use customer-facing-sdp

Reply:
customer-facing-sdp-id is removed, as in the base ns-nbi-yang model there is already a reference to tp-id under the SDP object.